flocker/node/agents/blockdevice.py

```diff
@@ -1456,6 +1507,18 @@ class BlockDeviceDeployer(PRecord):
     mountroot = field(type=FilePath, initial=FilePath(b"/flocker"))
 
     @property
+    def profiled_blockdevice_api(self):
+        """
+        Get an ``IProfiledBlockDeviceAPI`` provider which can create profiled
+        block devices for this deployer.
```

Comment:
"profiled block devices" is confusing - partly in a new way, partly because of existing naming confusing in this module.

What the thing can really create is volumes (and then it can attach them and expose the result via a Linux block device).

And "profiled" seems to hint at some kind of instrumented volume that will report back usage or performance information.

Maybe "create volumes configured based on pre-defined profiles" (though that's quite a mouthful; it would be great to have a convenient shorthand for this concept.  Maybe "profiled volume" is it, but maybe there's a less ambiguous option somewhere out there.)